portal/src/views/shared/Logo.vue

```diff
@@ -1,59 +1,27 @@
 <template>
-    <i id="header-logo" role="img" :aria-label="altText"></i>
+    <i id="header-logo" role="img" :aria-label="altText" class="icon" :class="iconClass"></i>
 </template>
 
 <script lang="ts">
 import Vue from "vue";
 
 export default Vue.extend({
     name: "Logo",
-    mounted(): void {
-        if (document.body.classList.contains("floodnet")) {
-            this.altText = this.$tc("layout.logo.floodnet.alt");
-        }
-    },
-    data(): {
-        altText: string;
-    } {
-        return {
-            altText: this.$tc("layout.logo.fieldkit.alt"),
-        };
+    computed: {
+        altText(): string {
```

Comment:
Anyway to do this in CSS? I'd like to avoid sprinkling "floodnet" all over the place.